src/components/medication-event-card.component.tsx

```diff
@@ -1,98 +0,0 @@
-import React from "react";
-import {
-  DosageInstruction,
-  MedicationDispense,
-  MedicationRequest,
-  Quantity,
-} from "../types";
-import styles from "./medication-event-card.scss";
-import {
-  getDosageInstruction,
-  getQuantity,
-  getRefillsAllowed,
-  getMedicationDisplay,
-  getMedicationReferenceOrCodeableConcept,
-} from "../utils";
-import { Tile, OverflowMenu } from "@carbon/react";
-import { useTranslation } from "react-i18next";
-import MedicationEventStatusTag from "./medication-event-status-tag";
-
-// can render MedicationRequest or MedicationDispense
-const MedicationEventCard: React.FC<{
-  medicationEvent: MedicationRequest | MedicationDispense;
-  actionMenu?: OverflowMenu;
-}> = ({ medicationEvent, actionMenu = null }) => {
-  const { t } = useTranslation();
-  const dosageInstruction: DosageInstruction = getDosageInstruction(
-    medicationEvent.dosageInstruction
-  );
-  const quantity: Quantity = getQuantity(medicationEvent);
-  const refillsAllowed: number = getRefillsAllowed(medicationEvent);
-
-  return (
-    <Tile className={styles.medicationEventTile}>
-      {actionMenu}
-      <div>
```

Comment:
Ugh, because I renamed, looks like we aren't getting a diff from Git, but rather it looks like the file has been deleted and created.  The main different in that the new "MedicationEvent" conmponent starts with this "div" here... the "Tile" and the "actionMenu" have been pulled out into the history-and-commments and prescription-details components.